core/templates/pages/splash-page/splash-page.component.html

```diff
@@ -1,174 +1,354 @@
-              ng-if="!$ctrl.userIsLoggedIn"
-              ng-click="$ctrl.onRedirectToLogin('/creator-dashboard?mode=create')"
-              class="btn oppia-splash-button oppia-splash-button-create oppia-splash-create-button-position oppia-transition-200"
-              translate="I18N_ACTION_CREATE_LESSON">
-      </button>
-      <button type="button"
-              ng-if="$ctrl.userIsLoggedIn"
-              ng-click="$ctrl.onClickCreateExplorationButton()"
-              class="btn oppia-splash-button oppia-splash-button-create oppia-splash-create-button-position oppia-transition-200"
-              translate="I18N_ACTION_CREATE_LESSON">
-      </button>
+      <form action="<[$ctrl.classroomUrl]>">
+        <button type="submit" ng-click="$ctrl.onClickBrowseLessonsButton()"
+                class="btn oppia-splash-button oppia-splash-button-browse oppia-splash-browse-button-position oppia-transition-200"
+                translate="I18N_ACTION_BROWSE_LESSONS">
+        </button>
+      </form>
     </div>
   </div>
-
-  <div class="oppia-splash-h2-container">
-    <h2 class="oppia-splash-h2 oppia-splash-h2-text" translate="I18N_SPLASH_SUBTITLE"></h2>
+  <div class="oppia-splash-section-one-right">
+    <img class="oppia-splash-main-image" ng-src="<[$ctrl.getMainBackgroundUrl()]>" alt="">
   </div>
 </div>
 
 <div class="oppia-splash-section-two">
-  <div class="oppia-splash-section-two-content oppia-splash-section-two-content-extra-info">
-    <div class="oppia-splash-bullet oppia-splash-bullet-extra-info">
-      <div class="oppia-splash-bullet-block oppia-splash-block-left-image">
-        <img ng-src="<[$ctrl.getStaticImageUrl('/splash/bullet1icon.svg')]>" class="oppia-splash-overlapping-image-1" alt="">
-      </div>
-      <div class="oppia-splash-bullet-block oppia-splash-block-right-text"
-           translate="I18N_SPLASH_FIRST_EXPLORATION_DESCRIPTION"
-           dir="auto">
-      </div>
+  <div class="oppia-splash-icon-1-container">
+    <img ng-src="<[$ctrl.getStaticImageUrl('/splash/userIcon.svg')]>" class="oppia-splash-icon-1" alt="">
+    <span class="splash-icon-text" translate="I18N_SPLASH_ICON_ONE_TEXT"></span>
+  </div>
+  <div class="oppia-splash-icon-2-container">
+    <img ng-src="<[$ctrl.getStaticImageUrl('/splash/languageIcon.svg')]>" class="oppia-splash-icon-2" alt="">
+    <span class="splash-icon-text" translate="I18N_SPLASH_ICON_TWO_TEXT"></span>
+  </div>
+  <div class="oppia-splash-icon-3-container">
+    <img ng-src="<[$ctrl.getStaticImageUrl('/splash/lessonIcon.svg')]>" class="oppia-splash-icon-3" alt="">
+    <span class="splash-icon-text" translate="I18N_SPLASH_ICON_THREE_TEXT"></span>
+  </div>
+</div>
+
+
+<div class="oppia-splash-section-three">
+  <div class="oppia-splash-section-three-left">
+    <h3 ng-if="$ctrl.isWindowNarrow()" translate="I18N_SPLASH_BENEFITS_TITLE"></h3>
+    <img ng-src="<[$ctrl.getStaticImageUrl('/splash/matthew_fractions.png')]>" class="splash-matthew-image" alt="">
```

Comment:
For non-SVG images use `<picture>` and provide WebP version of the image too. See [https://web.dev/serve-images-webp/](https://web.dev/serve-images-webp/) for reference.

Reply:
Done